### ISSUE TITLE
Script sync not functioning in Brave browser - websocket is connected

### ISSUE THREAD

**Issue by BenGitsCode** [arnoudkooi/sn-scriptsync#50]
I was having issues using scriptsync after not using it in a while and was able to isolate the problem to the Brave browser (it works fine when I switch to chrome). This is pretty low priority since I'm able to switch to chrome but I wanted to bring it to your attention (and also Brave is my browser of choice so I would like to help get it working there).

I'm including as much as I can think of as relevant but please let me know if I can provide more information.

SnUtils chrome extension 4.2.7.5
node v15.8.0
VS code 1.53.2
Brave Version 1.20.103 Chromium: 88.0.4324.152 (Official Build) (x86_64)
Helper consistently shows "WebSocket | Connected to VS Code ScriptScync WebSocket"

and when I try to activate it I get this error:
![Image 2021-02-18 at 9 18 16 AM](https://user-images.githubusercontent.com/16841950/108370073-416e1d00-71ca-11eb-840a-d4f3bc974d5a.jpg)

Here is the error logged to the console:

```js
sp_config?id=widget_editor&sys_id=35c4ae9b071e6010f3b8f2508c1ed09a&spa=1:1 Uncaught ReferenceError: g_form is not defined
    at XMLHttpRequest.client.onreadystatechange (chrome-extension://jgaodbdddndbaijmcljdbglhpdhnjobg/inject.js:2209)
client.onreadystatechange @ chrome-extension://jgaodbdddndbaijmcljdbglhpdhnjobg/inject.js:2209
XMLHttpRequest.send (async)
XMLHttpRequest.send @ js_includes_sp_libs.jsx:16520
postRequestToScriptSync @ chrome-extension://jgaodbdddndbaijmcljdbglhpdhnjobg/inject.js:2214
onclick @ sp_config?id=widget_editor&sys_id=35c4ae9b071e6010f3b8f2508c1ed09a&spa=1:1
```

Thank you in advance and please let me know how I can help further.

**Comment by arnoudkooi** [comment]
It does a post to localhost:1977 that gets blocked by the Brave Shield, seems you can disable it per instance by allowing Ads and Trackers (I'm not adding any :) )

![image](https://user-images.githubusercontent.com/840781/108375282-19021500-7202-11eb-84f2-169e19e02c4d.png)

**Comment by arnoudkooi** [comment]
Assuming this fixes it, please let me know.
